benches/guest_memory.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2020 Alibaba Cloud Computing. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
According to discussions in this [PR](https://github.com/rust-vmm/vm-memory/pull/27), shouldn't this be:

`// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause`?